examples/BioModels/1.ant

```diff
@@ -107,5 +107,14 @@ model *BIOMD0000000001()
   RemovalACh is "removal of ACh";
 
 
-  BLL identity "http://identifiers.org/chebi/CHEBI:27732"
+

```

Comment:
No changes should be made to the BioModels examples. You can create your own example files but don't modify the BioModels.